helix-view/src/editor.rs

```diff
@@ -1255,6 +1271,8 @@ impl Editor {
             handlers,
             mouse_down_range: None,
             cursor_cache: CursorCache::default(),
+            editor_clipping: ClippingConfiguration::default(),
+            user_defined_themes: Default::default(),
```

Comment:
Can we just modify the main theme loader? I'd like to avoid adding even more fields to `Editor`